src/librustc_errors/diagnostic_builder.rs

```diff
@@ -167,26 +170,51 @@ impl<'a> DiagnosticBuilder<'a> {
                                                   sp: S,
                                                   msg: &str)
                                                   -> &mut Self);
+
+    #[deprecated(note = "Use `span_suggestion_short_with_applicability`")]
     forward!(pub fn span_suggestion_short(&mut self,
                                           sp: Span,
                                           msg: &str,
                                           suggestion: String)
                                           -> &mut Self);
```

Comment:
While you are at it, fix indentation to new style (each arg on its own line, 4 spaces).